### PR TITLE
Care CARE Plus data ETL

### PR DESCRIPTION
**Pull request by hunterowens**
(need to figure out how to load socrata credentials)

### CODE REVIEW

**Comment by ian-r-rose** [comment]
I think there are two options:
1. Put the login/credentials in airflow variables.
1. Shove them into a connection that doesn't quite match Socrata, but has password/login fields that can be used for that purpose.

**Comment by hunterowens** [comment]
used airflow variables to match the 311 DAG.

**Comment by ian-r-rose** [comment]
Can you drop  1e2d6dc?

**Comment by hunterowens** [comment]
Done.

**Comment by hunterowens** [comment]
(rebasing is hard)

**Comment by ian-r-rose** [comment]
Looks good!

File: dags/public-health/care-311-to-postgres.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(days=1),
+}
+
+dag = DAG("care-plus-loader", default_args=default_args, schedule_interval="@daily")
+
+t1 = PythonOperator(
+    task_id='load-to-postgres',
+    provide_context=True
```
Comment (ian-r-rose) [inline dags/public-health/care-311-to-postgres.py]: Failure due to invalid syntax here.

**Pull request merged**